packages/cli-tools/src/client.ts

```diff
@@ -1,19 +1,31 @@
-import { StreamrClient, StreamrClientConfig } from '@streamr/sdk'
+import { KeyPairIdentityConfig, StreamrClient, StreamrClientConfig } from '@streamr/sdk'
 import merge from 'lodash/merge'
 import { Options } from './command'
 import { getConfig } from './config'
 
 export const getClientConfig = (commandOptions: Options, overridenOptions: StreamrClientConfig = {}): StreamrClientConfig => {
     const configFileJson = getConfig(commandOptions.config)?.client
     const environmentOptions: StreamrClientConfig = { environment: commandOptions.env }
-    const authenticationOptions: StreamrClientConfig = 
-        (commandOptions.privateKey !== undefined) ? { auth: { privateKey: commandOptions.privateKey } } : {}
+
+    const keyPairConfig: KeyPairIdentityConfig | undefined = 
+        (commandOptions.privateKey) ? { privateKey: commandOptions.privateKey } : undefined
+
+    if (keyPairConfig) {
+        keyPairConfig.publicKey = commandOptions.publicKey
+        keyPairConfig.keyType = commandOptions.keyType
+    }
```

Comment:
Could we set all these 3 values (`privateKey`, `publicKey` and `keyType`) at line 11? Or do we need both the ternary and the statement?